Public/Src/Cache/ContentStore/Distributed/NuCache/ClusterState.cs

```diff
@@ -130,5 +131,25 @@ public bool TryResolveMachineId(MachineLocation machineLocation, out MachineId m
         {
             return _idByLocationMap.TryGetValue(machineLocation, out machineId);
         }
+
+        /// <summary>
+        /// Gets a random locations excluding the specified location. Returns default if operation is not possible.

```

Comment:
you may use <inheritdoc /> here to avoid comments duplication, if you want.

Very minor.